packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -161,6 +161,17 @@ describe('plugin-meetings', () => {
       assert.isFalse(audio.isSelf());
     });
 
+    describe.only('#isLocallyMuted()', () => {
+      it('does not consider remote mute status - isLocallyMuted', async () => {
```

Comment:
the bit "- isLocallyMuted" can be removed now

Reply:
Removed :-)